contracts/DAO.sol

```diff
@@ -8,15 +8,12 @@ import "./ModuleBase.sol";
 
 /// @notice A minimum viable DAO contract
 contract DAO is IDAO, ModuleBase {
-    string public name;
-
     /// @notice Function for initializing the contract that can only be called once
     /// @param _accessControl The address of the access control contract
     /// @param _moduleFactoryBase The address of the module factory
     /// @param _name Name of the Dao
     function initialize(address _accessControl, address _moduleFactoryBase, string calldata _name) external initializer {
-        __initBase(_accessControl, _moduleFactoryBase);
-        name = _name;
+        __initBase(_accessControl, _moduleFactoryBase, _name);
```

Comment:
I guess I'm confused... what is the `_moduleFactoryBase` that's being initialized here?

Reply:
The module factories are maintaining the versions of the modules, so the modules need to know which factory they were deployed from.

However, in this case of the DAO contract, it doesn't really make sense, since the DAO Factory doesn't inherit from ModuleFactoryBase. So maybe we should rethink the DAO inheriting from ModuleBase, or figure out if the DAO factory can inherit from ModuleFactoryBase.